### ISSUE TITLE
The loss during the training increased. After the training of 10 epochs, the loss was about 1.1. Is this normal?Thanks for answering.

### ISSUE THREAD

**Issue by logicsh** [winterwindwang/Full-coverage-camouflage-adversarial-attack#16]
The loss during the training increased. After the training of 10 epochs, the loss was about 1.1. Is this normal?

**Comment by logicsh** [comment]
![texture2  40](https://user-images.githubusercontent.com/83349759/166884480-408bc7e7-3566-4559-94f8-f70e0f21d6af.png)
![1651737974(1)](https://user-images.githubusercontent.com/83349759/166884498-18eb16e3-51c6-42bd-814a-91f76fa86acb.png)
These are related pictures.

**Comment by winterwindwang** [comment]
> ![texture2 40](https://user-images.githubusercontent.com/83349759/166884480-408bc7e7-3566-4559-94f8-f70e0f21d6af.png) ![1651737974(1)](https://user-images.githubusercontent.com/83349759/166884498-18eb16e3-51c6-42bd-814a-91f76fa86acb.png) These are related pictures.

I never encounter this problem. I see you only use 1563 sample to train the adversarial texture, you can try use more training samples to train the texture.

**Comment by logicsh** [comment]
Thanks,I will try it.

**Comment by logicsh** [comment]
emmmmm...1563 is the quantity of batch , not the quantity of NPZ file. The batch_size is 8. So the number of training data is 1563x8=12504, which I think is probably correct. what do you think?

**Comment by logicsh** [comment]
Would you like to tell me your figure in this area is?thanks for answering!

**Comment by winterwindwang** [comment]
As show in here:https://winterwindwang.github.io/Full-coverage-camouflage-adversarial-attack/



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年5月6日(星期五) 中午12:45
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [winterwindwang/Full-coverage-camouflage-adversarial-attack] The loss during the training increased. After the training of 10 epochs, the loss was about 1.1. Is this normal?Thanks for answering. (Issue #16)





 
Would you like to tell me your figure in this area is?thanks for answering!

**Comment by winterwindwang** [comment]
I have updated the code repo, so I suggest you pull the repo and try the new repo.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "winterwindwang/Full-coverage-camouflage-adversarial-attack"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月6日(星期五) 中午12:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [winterwindwang/Full-coverage-camouflage-adversarial-attack] The loss during the training increased. After the training of 10 epochs, the loss was about 1.1. Is this normal?Thanks for answering. (Issue #16)





 
emmmmm...1563 is the quantity of batch , not the quantity of NPZ file. The batch_size is 8. So the number of training data is 1563x8=12504, which I think is probably correct. what do you think?

**Comment by logicsh** [comment]
okay,thanks!
